security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -13,6 +13,32 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-30"]
+== OpenShift File Integrity Operator 0.1.30
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.30:
+
+* link:https://access.redhat.com/errata/RHBA-2022:5538[RHBA-2022:5538 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
+
+[id="file-integrity-operator-0-1-30-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The File Integrity Operator is now supported on the following architectures:
++
+** IBM Power
+** IBM Z and LinuxONE
+
+[id="file-integrity-operator-0-1-30-bug-fixes"]
+=== Bug fixes
+
+* Previously, the File Integrity Operator used broad permissions coupled with missing labels, causing log warnings about excessive permissions granted to the Operator. Now, permissions are reduced where appropriate and labels are applied to the namespace, eliminating permission warnings in the log files. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2088201[*BZ#2088201*])
+
+* Previously, alerts issued by the File Integrity Operator did not set a namespace, making it difficult to understand where the alert originated. Now, the Operator sets the appropriate namespace, increasing understanding of the alert. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2101393[*BZ#2101393*])
+
+* Previously, the Operator Lifecycle Manager (OLM) deployed templates that assumed `openshift-file-integrity` namespace permissions for the File Integrity Operator, creating permissions when attempting to use the Operator outside the default namespace. Now, the deploy resources used by OLM are updated to use the correct namespace. This update fixes the permission issues and users can install the Operator in non-default namespaces. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2104897[*BZ#2104897*])
```

Comment:
I think this is a no-document one as well. Changes to let us install into a non-openshift* namespace came about after 0.1.24 (the bug was discovered on 0.1.26)